Order containers (#134)
commerce-docs/microsite-commerce-storefront: src/content/docs/dropins/order/containers/order-cost-summary.mdx

---
title: OrderCostSummary container
description: Learn about the OrderCostSummary container.
---

import Diagram from '@components/Diagram.astro';
import Callouts from '@components/Callouts.astro';
import OptionsTable from '@components/OptionsTable.astro';
import Aside from '@components/Aside.astro';
import { Steps } from '@astrojs/starlight/components';

> The `OrderCostSummary` container is used on Order Details and Return Details pages to display a breakdown of order costs. It includes the following sections:

Reviewer: ```suggestion
The `OrderCostSummary` container displays detailed order costs on the Order Details and Return Details pages. It includes the following sections:
```

- **Subtotal**: Displays the total cost of all items in the order before applying discounts, taxes, or additional charges.
- **Shipping**: Displays the shipping cost, which depends on the shipping method, location, and weight of the order.
- **Discount**: Displays any applicable discounts, such as promotional or volume-based offers, subtracted from the subtotal.
- **Coupon**: Displays the value of any applied coupons and their impact on the final cost.
- **Tax**: Displays the tax amount added to the order, calculated based on jurisdiction and item type.
- **Total**: Displays the final payable amount, including all adjustments such as discounts, shipping, and taxes.

If a value is not provided for any section (such as no discount or coupons are applied), the corresponding line is hidden. This ensures the container only displays relevant information.

> The display of tax amounts is configured at **Stores** > Configuration > **Sales** > **Tax** > **Order, Invoices, Credit Memos Display Settings**.

Reviewer: ```suggestion
The settings for displaying tax amounts can be configured at **Stores** > Configuration > **Sales** > **Tax** > **Order, Invoices, Credit Memos Display Settings**.
```

<Diagram caption="OrderCostSummary container">
  ![OrderCostSummary  container](@images/dropins/order/order-cost-summary.png)
</Diagram>

## Configurations

> The OrderCostSummary container provides the following configuration options:

Reviewer: ```suggestion
The `OrderCostSummary` container provides the following configuration options:
```
